locales/en/messages.json

```diff
@@ -4408,7 +4408,7 @@
         "message": "Set this to the highest incoming noise frequency that is needed to be controlled by the dynamic notch."
     },
     "pidTuningDynamicNotchCountHelp": {
-        "message": "Sets the number of dynamic notches per axis. With RPM filter enabled a value of 1 or 2 is recommended. Without RPM filter a value of 4 or 5 is recommended. Lower numbers will reduce filter delay, however it may increase motor temperature."
+        "message": "Sets the number of dynamic notches per axis. With RPM filter enabled a value of 1 or 2 is recommended. Without RPM filter a value of 4 or 5 is recommended. Lower numbers will reduce filter delay, however it may increase motor temperature.<br/></br/>For Cinelifters a value of 6 or 7 may be required where Dshot telemetry is not applicable."
```

Comment:
@nerdCopter not sure what you're saying here, but what I can tell you is that for really large lifters and x-class rigs, it is common to use APD 200a ESCs (and others) that are not RPM ready (and never will be). It is also common to use bi-blades on really large rigs, which themselves can generate 5 or more motor harmonics, and currently this leaves nothing for resonance peaks, not to mention that using DNs are not super effective without using more notches than harmonics present. So to have a couple more available when RPM is not an option is a good thing. 

My original suggestions was to keep the GUI limit to 5, and make the additional 2 notches only available via CLI, to avoid less experienced tuners for doing bad things. I still stand by this

Reply:
Okay, dropped the GUI changes for number of notches used.